app/models/offer.rb

```diff
@@ -6,6 +6,7 @@ class Offer < Listing
   scope :matched,   -> { includes(:matches_as_provider).references(:matches_as_provider).where.not(matches: {receiver_id: nil}) }
   scope :unmatched, -> { includes(:matches_as_provider).references(:matches_as_provider).where(matches: {receiver_id: nil}) }
   scope :matchable, -> { where(id: unmatched.pluck(:id) + inexhaustible.pluck(:id)) }
+  scope :in_service_areas, ->(ids) { where(service_area_id: ids) }
```

Comment:
- [ ] TODO: refactor this to the parent class, assuming that doesn't cause any problems